assets/global.js

```diff
@@ -59,7 +59,7 @@ function showRegion(country_alpha2, countryName, region, regionName, date) {
     else {
         $('#regionModalTitle').html(`${unescape(countryName)} / ${unescape(regionName)}`);
     }
-    $('#regionModalImage').attr('src', `data/${country_alpha2}/${date}/${region}/details.png`)
+    $('#regionModalImage').attr('src', `${window.location.origin}/data/${country_alpha2}/${date}/${region}/details.png`)
```

Comment:
`window.location.origin` resolves to `https://rtlive.de` even if the HTML is located in `https://rtlive.de/preview/index.html`.

At the same time it resolves to `http://localhost:4040` when the site is run locally for testing using https://github.com/michaelosthege/rtlive-dash-de/blob/master/nginx/start.sh

Reply:
What do you mean by "resolves to"? That it takes the files from the rt.live bucket?

Reply:
This line of JavaScript runs client-side in the browser: `window.location` is just the URL of the current browser tab.

Could be that I mixed up "resolves" and "evaluates"..